frontend/src/pages/Houses.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useLayoutEffect, useRef } from 'react';
```

Comment:
This whole file now lives in pages/Houses/, if we were to merge this, we'd all of a sudden have 2 of them with this being the older version.

Reply:
This change has already landed on master, so please git pull and remove this if necessary

Reply:
I did a pull from master and everything seems to be updated to the latest version.